src/selectors/roughHeights.js

```diff
@@ -183,6 +191,27 @@ function getScrollStateX(
   return ScrollbarState.HIDDEN;
 }
 
+/**
+ * @param {{
+ *   overflowX: string,
+ *   showScrollbarX: boolean,
+ * }} scrollFlags
+ * @param {number} height
+ * @param {number} scrollContentHeight
+ * @return {ScrollbarState}
+ */
+function getScrollStateY(scrollFlags, height, scrollContentHeight) {
```

Comment:
why can't scrollbarY's state not be `ScrollbarState.JOINT_SCROLLBARS` ?

Reply:
I am not sure what exactly `JOINT_SCROLLBARS` means here.  but if it means that _the vertical scroll will be shown if and only if the horizontal scrollbar is shown_, then i think we should not worry about this case  here as we have already subtracted `scrollbarXHeight` from `maxAvailableWidth` if horizontal scrollbar is visible before calling this method,  so it will return VISIBLE in that case too.

Reply:
> Could you point me to where this is happening?

https://github.com/schrodinger/fixed-data-table-2/blob/master/src/selectors/roughHeights.js#L122

Before calling getScrollStateY() we have already subtracted Horizontal scroll bar height i.e (scrollbarXHeight) from `maxAvailbaleHeight` which is passed as argument `height` in this method.   So this comparision `(scrollContentHeight > height)` in getScrollStateY()  will be true even if vertical scrollbar has to shown because of horizontal scrollbar (i.e JOINT_SCROLLBAR case).

Reply:
This is going to be hard to remember in the future. 
Do you think you can change this function to take in `roughAvailableHeight` instead?

You can then have this function check for the `JOINT_SCROLLBAR` scenario, and make it consistent with the other `getScrollStateX` function.

Reply:
Have made this consistent with getScrollStateX.